lib/database/models/logtags.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+module.exports = (sequelize) => {
+    const Sequelize = require('sequelize');
+
+    const LogTags = sequelize.define('LogTags', {
+        logId: {
+            type: Sequelize.INTEGER,
+        },
+        tagId: {
+            type: Sequelize.INTEGER,
+        },
+    });
+
+    LogTags.removeAttribute('id');
```

Comment:
I think this is not required if both logId and tagId have the `primary: true` flag

Reply:
Unfortunately it is required, otherwise I get the error `Unknown column 'id' in 'field list'` when requesting from this model. I do admit it's kind of a patchy fix, but that's all I could find